pkg/performanceprofile/controller/performanceprofile_controller.go

```diff
@@ -174,27 +154,45 @@ func (r *PerformanceProfileReconciler) removeOLMOperator() error {
 
 	for i := range csvs.Items {
 		csv := &csvs.Items[i]
-		if csv.Name == paoCSV {
-			klog.Infof("Removing performance-addon-operator CSV %s", paoCSV)
-			if err := r.Delete(context.TODO(), csv); err != nil {
-				return err
+		deploymentSpecs := csv.Spec.InstallStrategy.StrategySpec.DeploymentSpecs
```

Comment:
can we just check the deployment name? It should be hardcoded under the CSV, so it should be good enough for us.

Reply:
will look into that